src/main/java/ua/softserveinc/tc/service/impl/RoomServiceImpl.java

```diff
@@ -169,14 +169,47 @@ public Boolean isPossibleUpdate(BookingDto bookingDto) {
     }
 
     /**
+     * The method finds the maximum people in the room for period of time
+     * from dateLo to dateHi.
+     * @param dateLo start of period
+     * @param dateHi end of period
+     * @param bookings all reserved bookings in the time period
+     * @return The maximum number of people that are simultaneously in the room
+     */
+
+    private int maxRangeReservedBookings (Date dateLo, Date dateHi, List<Booking> bookings) {
+        if(dateLo == null)
+            throw new NullPointerException("dateLo is null");
+        else if(dateHi == null)
+            throw new NullPointerException("dateHi is null");
+        else if(bookings == null)
+            throw new NullPointerException("bookings is null");
+
+        long oneMinute = 60 * 1000;
```

Comment:
If we move this variable to class fields, we will clutter our class, it is not good to spirit of OOP. This variable neither represent a state of object, neither a state of class. This is just local variable with little life cycle. But I may do this variable final, but it not have big sense.